src/EventStore.Client.Streams/EventStoreClient.Read.cs

```diff
@@ -15,14 +16,14 @@
 #nullable enable
 namespace EventStore.Client {
 	public partial class EventStoreClient {
-		private ReadStreamResult ReadAllAsync(Direction direction, Position position, long maxCount,
+		private ReadAllStreamResult ReadAllAsync(Direction direction, Position position, long maxCount,
```

Comment:
**NOTE:** This will be a binary breaking change, that should be reflected in the package number.